geode-management/src/main/java/org/apache/geode/management/configuration/AbstractConfiguration.java

```diff
@@ -17,15 +17,19 @@
 
 import static org.apache.commons.lang3.StringUtils.isBlank;
 
-import java.io.Serializable;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.commons.lang3.StringUtils;
 
 import org.apache.geode.annotations.Experimental;
 import org.apache.geode.lang.Identifiable;
 import org.apache.geode.management.api.JsonSerializable;
+import org.apache.geode.management.runtime.RuntimeInfo;
 
 @Experimental
-public abstract class AbstractConfiguration
-    implements Identifiable<String>, Serializable, JsonSerializable {
+public abstract class AbstractConfiguration<R extends RuntimeInfo>
```

Comment:
please javadoc the generic type param.  see ClusterManagementOperation for a comparable example

Reply:
done